### PR TITLE
Vents and Scrubbers can be placed 180 degrees on each other, with a max of 4 total each tile

### LINKED ISSUE #15713: Atmos pipe devices take up multiple directions.
**Problem Description**:
What is the problem?
vents and scrubbers take up 2 directions on a tile instead of 1. Example: A vent piped going up also takes up the down pipe position.
**What did you expect to happen**: 
Why do you think this is an issue?
They should only take up 1 position, allowing you to wrench pipes and pipe devices in the opposite direction on the tile as well.
**What happened instead**:
How is what happened different from what you expected?
You can only put 1 vent/scrubber facing up/down, and one facing left/right
**Why is this bad/What are the consequences:**
Why do you think this is an important issue?
They used to be set so that you can for example place 4 vents facing 4 different directions on a single tile.
**Steps to reproduce the problem**:
The most important section. Review everything you did leading up to causing the issue.
Place a vent/scrubber in one direction and wrench it. Place another in the opposite direction and try to wrench it.
**When did the problem start happening**:
If your report is about something that used to work but no longer does, when was the last time you remember it working?

It worked when I used to make my turbine TEG setups, I used single tiles with 3 vents and a scrubber on it to bypass pressure restrictions that volume pumps had.

dreamseeker_NCkO9t7odf
dreamseeker_3xd54EEDwC

### PR DESCRIPTION
**Pull request by Kepplerx**
Fixes an unintended sideeffect of #9951
Fixes #15713 

## What Does This PR Do

Vents and scrubbers can now be on the same tile, opposed to each other, instead of max 2 on a tile

## Why It's Good For The Game

This was a unintended change, this would revert that, makes many Atmos related constructs more viable, and possible to make.

## Images of changes

image

## Changelog
:cl:
fix: Fixed vents not being able to be put 180 degrees on each other
/:cl:

### CODE REVIEW

File: code/game/machinery/pipe/construction.dm [hunk 1]
```diff
@@ -256,8 +256,10 @@
 			return dir|flip
 		if(PIPE_SIMPLE_BENT, PIPE_INSULATED_BENT, PIPE_HE_BENT, PIPE_SUPPLY_BENT, PIPE_SCRUBBERS_BENT)
 			return dir //dir|acw
-		if(PIPE_CONNECTOR, PIPE_UVENT, PIPE_PASV_VENT, PIPE_SCRUBBER, PIPE_HEAT_EXCHANGE, PIPE_INJECTOR)
+		if(PIPE_CONNECTOR,  PIPE_HEAT_EXCHANGE, PIPE_INJECTOR)
 			return dir|flip
+		if(PIPE_UVENT, PIPE_PASV_VENT, PIPE_SCRUBBER)
+			return dir
```
Comment (SteelSlayer) [inline code/game/machinery/pipe/construction.dm:262]: Can you not put these inside the `if` on line 257?

Comment (Kepplerx) [reply code/game/machinery/pipe/construction.dm:262]: Why not indeed, let me actually go ahead and do that

Comment (Kepplerx) [reply code/game/machinery/pipe/construction.dm:262]: Oh, it was to make clear the difference between 257 which was bent pipes, and how they also are applied the acw/cw, and vents and scrubbers which do not @SteelSlayer

Comment (SteelSlayer) [reply code/game/machinery/pipe/construction.dm:262]: I'm really just referring to the return values. Both the new `if` you've added and the check on 257 does a `return dir`, therefore they should probably be combined.

**Comment by warior4356** [comment]
This is really going to be abused in the SM.

**Comment by DinBlade** [comment]
> This is really going to be abused in the SM.

Rad collectors are meh at best, so unless someone is doing Rad related antaging or delams, SM is not the issue this makes. TEGs tho and what little I know of them, this is a gamechanger.

**Comment by DesolateG** [comment]
> > This is really going to be abused in the SM.
> 
> Rad collectors are meh at best, so unless someone is doing Rad related antaging or delams, SM is not the issue this makes. TEGs tho and what little I know of them, this is a gamechanger.

It's not anything new, just a fix of a bug that no one noticed until I tried making one of my old TEG setups.

**Comment by Fox-McCloud** [comment]
This seems unwise to do, at a glance.

**Comment by DesolateG** [comment]
> This seems unwise to do, at a glance.

Could you clarify?

**Comment by Arvenius169** [comment]
I was always under the impression that this had been an intentional decision to reduce crazy atmos device spam.

**Comment by DesolateG** [comment]
> I was always under the impression that this had been an intentional decision to reduce crazy atmos device spam.

It was a bug caused by Mitch's PR.

**Comment by Arvenius169** [comment]
> This is really going to be abused in the SM.

SM almost never has the draft touched. I don't think that's a legitimate worry.

File: code/game/machinery/pipe/construction.dm (hunk 1 above)
Comment (variableundefined) [reply code/game/machinery/pipe/construction.dm:262]: Probably won't make a difference when it comes to performance.

**Review by variableundefined — APPROVED** [review]

**Pull request merged**